src/open_inwoner/search/views.py

```diff
@@ -62,10 +64,9 @@ def search(self, form):
             self.log_user_action(user, _("search query: {query}").format(query=query))
 
         # Check if the query exactly matches with a case that belongs to the user
-        # TODO should be implemented for KVK as well
-        if hasattr(self.request.user, "bsn"):
+        if search_params := get_fetch_parameters(self.request):
```

Comment:
Here you see it's unclear: this makes get_fetch_parameters() look like magic. We're getting the fetch params for the user, not a generic mystery search.